Assets/HoloToolkit/ARCA/Plugins/WSA/x86/OpenCVWrapper.dll.meta

```diff
@@ -0,0 +1,113 @@
+fileFormatVersion: 2
```

Comment:
Where does this DLL come from? Is it from source you wrote? If so, that source needs to be submitted to https://github.com/Microsoft/MixedRealityToolkit where Microsoft can then build and sign the binary for inclusion here.

Reply:
This is a custom dll.  I've now submitted another PR for this here https://github.com/Microsoft/MixedRealityToolkit/pull/154